tests/warehouses/sqlserver_suite copy.py

```diff
@@ -0,0 +1,18 @@
+#  Copyright 2020 Soda
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#   http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from tests.common.sql_test_case import TARGET_SQLServer
+from tests.common.sql_test_suite import SqlTestSuite
+
+class SQLServerSuite(SqlTestSuite):
```

Comment:
Why is this file called `sqlserver_suite copy.py` ?
Is that a glitch or on purpose?

Reply:
I think a typo, I’ll change it